### PR TITLE
Cards

### PR DESCRIPTION
**Pull request by vijaytembugade**
Live Preview Link : https://vijaytembugade.github.io/XenonUI-Component-Library/Components/Cards/cards.html

Added Card and its types.
- Basic Card
- Card with text Overlay
- Card with dismis
- Card with badge
- Horizontal/Verticle Card
- Card with header
- Text only Card

### CODE REVIEW

File: Components/Cards/cards.css [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+  transition: all 0.5s ease;
+}
+.card:hover {
+  cursor: pointer;
+  box-shadow: rgba(0, 0, 0, 0.25) 0px 14px 28px,
+    rgba(0, 0, 0, 0.22) 0px 10px 10px;
+}
+
+/* overlay on card */
+.overlay-content {
```
Comment (sakleprajwal) [inline Components/Cards/cards.css:23]: You can use shorthand property for position as below
position: absolute 10px 50px 50px 100px;

File: Components/Cards/cards.css [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+  font-weight: 700;
+}
+.card:hover .overlay-content {
+  opacity: 0.95;
+}
+
+/* card header */
+
+.card-header {
+  padding: 1rem 0.5rem 0.5rem;
```
Comment (sakleprajwal) [inline Components/Cards/cards.css:49]: Don't use rem unit for the sizes that are less than 1rem

**Review by horsemaker — APPROVED** [review]
The overlay card kinda scared me! 👀 jk
Looks great!

**Review by mimansha-swarup — APPROVED** [review]
looks good to go

**Review by MarkVed17 — APPROVED** [review]
These cards look really nice. On the code syntax part, just avoid using rem for anything less than 10px.

File: Components/Cards/cards.css (hunk 1 above)
Comment (vijaytembugade) [reply Components/Cards/cards.css:23]: Oh. Thanks. I will make this changes.

File: Components/Cards/cards.css (hunk 2 above)
Comment (vijaytembugade) [reply Components/Cards/cards.css:49]: Yeah. I have been using them meanwhile. I will try not to use it explicitaly.

**Comment by preethasri** [comment]
hey vijay the cards are awesome . just a suggestion can you add the titles to each card instead of putting all of them at top

**Comment by KaustubhM1997** [comment]
Looks great. That extra space between the image and the card's border looks neat!

**Pull request merged**